### PR TITLE
[TD-13703]<docs>: add schemaless writing description

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

File: documentation20/cn/08.connector/01.java/docs.md [hunk 1]
```diff
@@ -600,6 +600,43 @@ public void setString(int columnIndex, ArrayList<String> list, int size) throws
...
 
+### <a class="anchor" id="schemaless_java"></a>无模式写入
+
+从 2.2.0.0 版本开始，TDengine 增加了对无模式写入功能。无模式写入兼容 InfluxDB 的 行协议（Line Protocol）、OpenTSDB 的 telnet 行协议和 OpenTSDB 的 JSON 格式协议。详情请参见[无模式写入](https://www.taosdata.com/docs/cn/v2.0/insert#schemaless)。
+注意：
+* JDBC-RESTful 实现并不提供无模式写入这种使用方式
+* 以下示例代码基于taos-jdbcdriver-2.0.36
+
+示例代码：
+```java
```
Comment (sangshuduo) [inline documentation20/cn/08.connector/01.java/docs.md:616]: need a blank line before code block

File: documentation20/cn/08.connector/01.java/docs.md [hunk 2]
```diff
@@ -600,6 +600,43 @@ public void setString(int columnIndex, ArrayList<String> list, int size) throws
...
+            stmt.executeUpdate("drop database if exists test_schemaless");
+            stmt.executeUpdate("create database if not exists test_schemaless");
+            stmt.executeUpdate("use test_schemaless");
+        }
+    }
+}
+```
+
 ### <a class="anchor" id="set-client-configuration"></a>设置客户端参数
 从TDengine-2.3.5.0版本开始，jdbc driver支持在应用的第一次连接中，设置TDengine的客户端参数。Driver支持JDBC-JNI方式中，通过jdbcUrl和properties两种方式设置client parameter。
```
Comment (sangshuduo) [inline documentation20/cn/08.connector/01.java/docs.md:647]: need a blank line between title and text

File: documentation20/en/08.connector/01.java/docs.md [hunk 3]
```diff
@@ -578,6 +578,43 @@ public void setShort(int columnIndex, ArrayList<Short> list) throws SQLException
 public void setString(int columnIndex, ArrayList<String> list, int size) throws SQLException
 public void setNString(int columnIndex, ArrayList<String> list, int size) throws SQLException
 ```
+
+### Data Writing via Schemaless
+Starting with version 2.2.0.0, TDengine supports schemaless function. schemaless writing protocol is compatible with InfluxDB's Line Protocol, OpenTSDB's telnet and JSON format protocols, Please see [Schemaless Writing](https://www.taosdata.com/docs/en/v2.0/insert#schemaless)
```
Comment (sangshuduo) [inline documentation20/en/08.connector/01.java/docs.md:584]: same here

File: documentation20/en/08.connector/01.java/docs.md [hunk 4]
```diff
@@ -578,6 +578,43 @@ public void setShort(int columnIndex, ArrayList<Short> list) throws SQLException
...
+Note:
+* Jdbc-restful implementations do not provide Schemaless-Writing
+* The following sample code is based on taos-jdbcdriver-2.0.36
+
+Sample Code:
+```java
```
Comment (sangshuduo) [inline documentation20/en/08.connector/01.java/docs.md:592]: blank line

File: documentation20/en/08.connector/01.java/docs.md [hunk 5]
```diff
@@ -578,6 +578,43 @@ public void setShort(int columnIndex, ArrayList<Short> list) throws SQLException
...
+            stmt.executeUpdate("drop database if exists test_schemaless");
+            stmt.executeUpdate("create database if not exists test_schemaless");
+            stmt.executeUpdate("use test_schemaless");
+        }
+    }
+}
+```
+
 ### Set client configuration in JDBC
 Starting with TDengine-2.3.5.0, JDBC Driver supports setting TDengine client parameters on the first connection of a Java application. The Driver supports jdbcUrl and Properties to set client parameters in JDBC-JNI mode.
```
Comment (sangshuduo) [inline documentation20/en/08.connector/01.java/docs.md:623]: need a blank line

File: documentation20/cn/08.connector/01.java/docs.md (hunk 1 above)
Comment (huolibo) [reply documentation20/cn/08.connector/01.java/docs.md:616]: add blank line as you see, but I wonder this is for beauty or format requirements?

Comment (sangshuduo) [reply documentation20/cn/08.connector/01.java/docs.md:616]: It's Markdown standard. You can use mdl to check it.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**